sdks/java/extensions/avro/src/main/java/org/apache/beam/sdk/extensions/avro/schemas/utils/AvroUtils.java

```diff
@@ -161,6 +152,25 @@ public class AvroUtils {
       new ForLoadedType(ReadableInstant.class);
   private static final ForLoadedType JODA_INSTANT = new ForLoadedType(Instant.class);
 
+  public static void addLogicalTypeConversions(final GenericData data) {
```

Comment:
Wondering, should these conversions all be singletons?

Reply:
This looks do be the the default usage in avro libs.